superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -224,9 +224,12 @@ export class Tabs extends React.PureComponent {
       title: t('Delete dashboard tab?'),
       content: (
         <span>
-          Deleting a tab will remove all content within it. You may still
-          reverse this action with the <b>undo</b> button (cmd + z) until you
-          save your changes.
+          {t(
```

Comment:
I wonder if template literals would make this block a little more legible/parsable. Not a dealbreaker.